### ISSUE TITLE
IdsMultiSelect: Accessibility WCAG Guideline: 4.1.2 Name, Role, Value Defect

### ISSUE THREAD

**Issue by CindyMercadoReyes** [infor-design/enterprise-wc#1232]
**Describe the bug**
[Global] Role of options present inside combobox are not defined.

**To Reproduce**

WCAG Guideline: 4.1.2 Name, Role, Value

"1. Launch the URL: https://main.wc.design.infor.com/ids-multiselect/example.html
2. Navigate to options present inside combobox using arrow keys and verify"
3. Role of options present inside combobox are not defined. Screen reader reads it as plain text.

**Expected behavior**
Role of options present inside combobox should be defined as checkbox.

User Impact: Without proper role screen reader user will not be able to understand the purpose and functionality of element.

**Screenshots**
https://www.screencast.com/t/3gHWrdlL

Dev Recommendation: Provide aria role as checkbox for the element.

**Platform**
Win 10/ Chrome/ NVDA

**Additional context**

**Comment by tmcconechy** [comment]
@CindyMercadoReyes I think that one issue is that im not seeing `aria-selected` on the selected item. The roles are there but not a checkbox role.  Lets see if aria-selected needs to be added on initial load and test this with the screen reader further
